### DISCUSSION TITLE
Prusa MK3S+ suddenly does not recognize pause without firware or software changes.

### DISCUSSION THREAD

**Discussion by HappyOldtimer** [smartin015/continuousprint#235] [Q&A]
Hi,

The issue I have is a bit strange from my perspective: I have four MK3s running, and only one of them had an issue when I set a job which would either run several times or print multiple items.  I only do manual clearing of the bed, by the way.
Behavior: prints just fine, success print gcode runs fine, and then it doesn't show 'resume print' on the display of the MK3. Resume in octoprint interface works just fine.
Behavior of the other three printers: prints run just fine, and after removing the print the display on the MK3 says 'resume print'.

But since then (couple of months ago), all four printers have started the behavior where no 'resume' is displayed on the MK3, while the resume in the octoprint interface works just fine. And I did not change anything in the meantime before and after the behavior change.

I just don't get it. I've already uninstalled continuous print and reinstalled, but that makes no difference. 
I have two different firmware versions running on the MK3s, so that doesn't seem to be the issue. 
The printers have been completely off and on, and so have the OrangePi's I'm running octoprint on. 
Upgrading octoprint also didn't change anything, by the way. 

Would anybody have any ideas as to what the issue might be? I can't wrap my head around why it would suddenly stop.

Regards,
Paul

**Comment by smartin015** [comment]
Hey Paul - not sure if you've found a fix to your problem yet, but if you attach a sysinfo bundle I could take a look at the logs to see if there's anything out of place there. Just follow the [bug report](https://github.com/smartin015/continuousprint/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=) template to do that. Thanks!

**Reply by HappyOldtimer** [reply]
Hi Scott,

I appreciate you taking the time to ask :-) No, I have not found a solution yet, I'm afraid...
Attached is my system bundle, anything you can find would be a bonus!

Kind regards,
Paul

[octoprint-systeminfo-20230817180813.zip](https://github.com/smartin015/continuousprint/files/12372429/octoprint-systeminfo-20230817180813.zip)
